src/pages/Committees.js

```diff
@@ -285,28 +340,33 @@ function Datasladden() {
 function DEMON() {
     return (
         <p>
-            Interested in music? DEMON (Swedish for 'CS and Media's orchestra club') is the joined committee 
+            Interested in music? DEMON (Swedish for "CS and Media's orchestra club") is the joined committee
             for music. They do not have a fixed repetition place, but can often be found at Hötorget. 
             There instruments can be freely borrowed, including a few guitars, a digital drum set and a digital 
-            piano. The head of committee is the ÄrkeDEMON (mail: arkedemon@datasektionen.se).
+            piano. The head of committee is the ÄrkeDEMON.
             <br/>
             To get in the committee, try asking people in META to get access to their Slack.
```

Comment:
This is wrong for several reasons. They don't have a Slack, they use Discord, and the correct way is to email ÄrkeDEMON, not randomly try to guess who he is in META

Reply:
The Slack info is from [Datas website](https://datasektionen.se/namnder/demon?lang=en), and randomly asking someone in META is for "Ask the ÄrkeDEMON or somebody else who you think might be in it if you want to get an invite." (same page). I'll update this for our website.